cmd/system-probe/api/server.go

```diff
@@ -46,6 +49,11 @@ func StartServer(cfg *config.Config) error {
 
 	mux.Handle("/debug/vars", http.DefaultServeMux)
 
+	// Expose telemetry endpoint
+	if ddconfig.Datadog.GetBool("telemetry.enabled") {
```

Comment:
Can we make this a field on the system-probe config object, and read that here instead?

Reply:
Done